app/assets/src/components/views/bulk_download/ChooseStep.jsx

```diff
@@ -395,10 +416,74 @@ class ChooseStep extends React.Component {
     );
   };
 
-  render() {
-    const { onContinue, selectedSampleIds } = this.props;
+  renderInvalidSamplesWarning = () => {
+    const { invalidSampleNames } = this.props;
 
-    const numSamples = selectedSampleIds.size;
+    const header = (
+      <div className={cs.header}>
+        <span className={cs.highlight}>
+          {invalidSampleNames.length} sample
+          {invalidSampleNames.length > 1 ? "s" : ""} won't be included in the
+          bulk download
+        </span>, because they are in progress or failed samples:
+      </div>
+    );
+
+    const content = (
+      <div className={cs.messageContainer}>
+        {invalidSampleNames.map((name, index) => {
+          return (
+            <div key={index} className={cs.messageLine}>
+              {name}
+            </div>
+          );
+        })}
+      </div>
+    );
+
+    return (
```

Comment:
What happens if all selected samples end up being invalid? Perhaps we should show an error and not allow the user to create the bulk download. (perhaps even hide the Continue button)

Reply:
Done